webhooks-extension/docs/InstallPrereqs.md

```diff
@@ -0,0 +1,88 @@
+# Prerequisites
+
+1. [Cluster requirements](#cluster-requirements)
+2. [Install prereqs](#install-prereqs)
+3. [Knative setup](#domain-setup-for-knative-serving)
+
+## Cluster requirements
+
+**All Kubernetes Clusters:**
+
+- Knative requires a Kubernetes cluster running version v.1.11 or greater.
+- Cluster must also be supplied with sufficient resources, for a single node cluster _(6 CPUs, 10GiB Memory & 2.5GiB swap)_.
+
+**Docker Desktop Only:**
+
+- Currently only runs Kuberenetes v1.11 in it's **edge** version.
+
+
+## Install prereqs
+
+1. Install [Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) 
+
+2. Install [Tekton Dashboard](https://github.com/tektoncd/dashboard)
+
+3. Install [Istio](https://istio.io/docs/setup/kubernetes/install/)
+
+   - **For a quickstart istio install:**
+ 
+       - Clone this repository: `git clone https://github.com/tektoncd/experimental.git`
+       - Run the istio installation script: `./scripts/install_istio.sh <version>` 
+         
+         **Note:** requires Helm to be installed and recommend using 1.1.7 as `<version>`
```

Comment:
I'd clarify that it's Istio 1.1.7 and not Helm 1.1.7, any Helm 2.x version should do (we could recommend 2.7.x or greater, that's what I've had success with - this requirement really isn't cool btw but it's here now so not to be fixed as part of this PR!)

Reply:
requires Helm 2.x to be installed and recommend using Istio 1.1.7